backend/data/src/main/kotlin/io/tolgee/formats/android/out/AndroidStringsXmlFileWriter.kt

```diff
@@ -25,6 +21,10 @@ class AndroidStringsXmlFileWriter(private val model: AndroidStringsXmlModel) {
   private fun Element.addToElement(unit: Map.Entry<String, AndroidXmlNode>) {
     when (val node = unit.value) {
       is StringUnit -> {
+        val comment = node.comment
+        if (!comment.isNullOrEmpty()) {
```

Comment:
It wold be great to Create a private extension function Element.comment, which will add the comment only if it's not null to remove the deduplicate the duplicated null checking logic.